Sources/StreamVideoSwiftUI/CallView/ScreenSharing/ScreenSharingView.swift

```diff
@@ -7,63 +7,63 @@ import SwiftUI
 
 public struct ScreenSharingView<Factory: ViewFactory>: View {
 
+    @Injected(\.colors) var colors
+
     @ObservedObject var viewModel: CallViewModel
     var screenSharing: ScreenSharingSession
-    var availableFrame: CGRect
+    var frame: CGRect
+    var innerItemSpace: CGFloat
     var viewFactory: Factory
+    var isZoomEnabled: Bool
 
-    private let thumbnailSize: CGFloat = 120
-    private var thumbnailBounds: CGRect {
-        CGRect(x: 0, y: 0, width: thumbnailSize, height: thumbnailSize)
-    }
+    private let identifier = UUID()
 
     public init(
         viewModel: CallViewModel,
         screenSharing: ScreenSharingSession,
         availableFrame: CGRect,
-        viewFactory: Factory = DefaultViewFactory.shared
+        innerItemSpace: CGFloat = 8,
+        viewFactory: Factory = DefaultViewFactory.shared,
+        isZoomEnabled: Bool = true
     ) {
         self.viewModel = viewModel
         self.screenSharing = screenSharing
-        self.availableFrame = availableFrame
+        self.frame = availableFrame
+        self.innerItemSpace = innerItemSpace
         self.viewFactory = viewFactory
+        self.isZoomEnabled = isZoomEnabled
     }
 
     public var body: some View {
-        VStack(alignment: .leading) {
+        VStack(spacing: innerItemSpace) {
             if !viewModel.hideUIElements {
                 Text("\(screenSharing.participant.name) presenting")
-                    .foregroundColor(.white)
+                    .foregroundColor(colors.text)
```

Comment:
Did you check this in action? Since the background was black, the text had to be white.